modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java

```diff
@@ -202,6 +202,10 @@
     public static final String ENUM_PROPERTY_NAMING = "enumPropertyNaming";
     public static final String ENUM_PROPERTY_NAMING_DESC = "Naming convention for enum properties: 'camelCase', 'PascalCase', 'snake_case', 'UPPERCASE', and 'original'";
 
+    public static final String SERIALIZATION_LIBRARY = "serializationLibrary";
```

Comment:
This is great, once merged I will remove `org.openapitools.codegen.languages.JavaClientCodegen.SERIALIZATION_LIBRARY` and use that instead.